src/index.ts

```diff
@@ -74,6 +74,8 @@ function activateFileBrowser(app: JupyterLab, manager: IDocumentManager, factory
   // Fetch the initial state of the settings.
   Promise.all([settingRegistry.load(PLUGIN_ID), app.restored])
   .then(([settings]) => {
+    const gitHubBaseUrl = settings.get('gitHubBaseUrl').composite as string;
+    gitHubBrowser.gitHubBaseUrl = gitHubBaseUrl;
```

Comment:
And here we can check if it is an invalid value coming out of the settings system. For brevity, you can set it like
```typescript
gitHubBrowser._gitHubBaseUrl = gitHubBaseUrl | DEFAULT_GITHUB_BASE_URL;
```

Reply:
That syntax doesn't seem to work? 
```
src/index.ts(78,5): error TS2322: Type 'number' is not assignable to type 'string'.
src/index.ts(78,30): error TS2362: The left-hand side of an arithmetic operation must be of type 'any', 'number' or an enum type.
src/index.ts(78,30): error TS2533: Object is possibly 'null' or 'undefined'.
src/index.ts(78,40): error TS2363: The right-hand side of an arithmetic operation must be of type 'any', 'number' or an enum type.
```
i.e. it appears to want both sides of the `|` operator be be numbers?

Reply:
Yep, typo on my part.